e2e/portal/tests/Navigation/BasicNavigation.spec.ts

```diff
@@ -14,56 +14,73 @@ import RegistrationsPage from '@121-e2e/portal/pages/RegistrationsPage';
 
 const projectId = 2;
 const projectTitle = 'NLRC Direct Digital Aid Program (PV)';
+// Arrange
+const reset = async () => {
```

Comment:
Why pull `reset` and `login` into separate functions? This seems like a departure from the pattern that we have in all the other tests